about/index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About | Doodle Dandy</title>
+    <link rel="stylesheet" href="../assets/vendor/simplecss/simple.min.css" />
+    <link
+      rel="stylesheet"
+      href="../assets/vendor/fontawesome/6.2.1/css/fontawesome.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../assets/vendor/fontawesome/6.2.1/css/brands.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../assets/vendor/fontawesome/6.2.1/css/solid.min.css"
+    />
+    <link rel="stylesheet" href="../assets/styles/main.css" />
+    <link
+      rel="shortcut icon"
+      href="../assets/images/favicon.ico"
+      type="image/x-icon"
+    />
+  </head>
+
+  <body>
+    <header>
+      <img src="../assets/images/header.png" alt="Doodle Dandy" />
+      <nav>
+        <a href="/">Home</a>
+        <a href="/bernedoodle">Bernedoodles</a>
+        <a href="/aussiedoodle">Aussiedoodles</a>
+        <a href="/about" class="available">About us</a>
+        <a href="/available">Available Puppies</a>
+        <a href="/reviews">Reviews</a>
+        <a href="/contact">Contact us</a>
+      </nav>
+    </header>
+    <main>
+      <h1>About us</h1>
+<p></p>
+
+<p>
+
+We are not your average breeder! We focus on creating an experience for the families and 
+puppies that will bring joy for years to come.
+<p></p>
+
+How are we different?
+
+<ul>
+  <li><i class="fa-solid fa-paw"></i>&nbsp;&nbsp;Gift Bag: We send home a gift bag that includes the popular Snuggle Puppy and many other goodies to help your pup transition to your home</li>
+  <li><i class="fa-solid fa-paw"></i>&nbsp;&nbsp;Communication: Once your adoption application is received, we will provide photos, video, and even Facetime with the puppy as we approach adoption day</li>
+<li><i class="fa-solid fa-paw"></i>&nbsp;&nbsp;Education: We have a ‘Congrats file’ that’s full of great information, including:</li>
+ <li class="indent"> What is the best way to travel with my puppy?</li>
+ <li class="indent"> What do I need to know about the drive/flight home?</li>
+ <li class="indent"> We share tips to make the transition to home as smooth as possible while also creating good long-term habits</li>
+ <li class="indent"> Grooming guidelines</li>
+ <li class="indent"> Where do I start with toys?</li>
+ <li class="indent"> Why crate train?</li>
```

Comment:
You need this list to be in a new `ul` element with the `indent` class.

Like this:

```suggestion
  <ul class="indent">
    <li> What is the best way to travel with my puppy?</li>
    <li> What do I need to know about the drive/flight home?</li>
    <li> We share tips to make the transition to home as smooth as possible while also creating good long-term habits</li>
    <li> Grooming guidelines</li>
    <li> Where do I start with toys?</li>
    <li> Why crate train?</li>
  </ul>
```

Reply:
Also the line about `We share tips to make the transition to home as smooth as possible while also creating good long-term habits` doesn't make sense in the list.